### PR TITLE
DEV added not finishet yet task 10 (Week 2)

### PR DESCRIPTION
**Pull request by JustynaGaryga**
I added task 10 done in half. I am going to finish it tomorrow.

### CODE REVIEW

File: Week2task10/src/SubmitButton.java [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+		frame.setLayout(new BorderLayout());
+		frame.setVisible(true);
+		
+		// the text field
+		JPanel panel = new JPanel();
+		JTextField nameField = new JTextField(30);
+		JTextField surnameField = new JTextField(30);
+		JTextField emailField = new JTextField(30);
+		JLabel name = new JLabel("Name: ");
+	    JLabel surname = new JLabel("Surname: ");
```
Comment (AndreeaMartinovici) [inline Week2task10/src/SubmitButton.java:39]: Please add a space or tab here that this line be aligned with the line above.

File: Week2task10/src/SubmitButton.java [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+	        }
+		});
+		
+		//after clicking button, window show up displaying the texts that have beed added in the 
+		//text fields
+		submit.addActionListener(new ActionListener() {
+			@Override
+			public void actionPerformed(ActionEvent arg0) {
+				// TODO Auto-generated method stub
+				JPanel buttonPanel = new JPanel();
```
Comment (AndreeaMartinovici) [inline Week2task10/src/SubmitButton.java:94]: Please make sure that all lines are aligned here as well.

**Review by AndreeaMartinovici — CHANGES_REQUESTED** [review]

File: Week2task10/src/SubmitButton.java [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+	        	}
+	        }
+		});
+		surnameField.addKeyListener(new KeyAdapter() {
+	        public void keyReleased(KeyEvent e) {
+	        	if (nameField.getText().length() == 0 || surnameField.getText().length() == 0 
+		        		|| emailField.getText().length() == 0) {
+	        				submit.setEnabled(false);
+	        	} else {
+	        	submit.setEnabled(true);
```
Comment (AndreeaMartinovici) [inline Week2task10/src/SubmitButton.java]: Add a space or tab here as well.

File: Week2task10/src/SubmitButton.java [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+	        	}
+	        }
+		});
+		emailField.addKeyListener(new KeyAdapter() {
+	        public void keyReleased(KeyEvent e) {
+	        	if (nameField.getText().length() == 0 || surnameField.getText().length() == 0 
+		        		|| emailField.getText().length() == 0) {
+	        				submit.setEnabled(false);
+	        	} else {
+	        	submit.setEnabled(true);
```
Comment (AndreeaMartinovici) [inline Week2task10/src/SubmitButton.java]: Here as well please add a space or tab.

File: Week2task10/src/SubmitButton.java (hunk 1 above)
Comment (JustynaGaryga) [reply Week2task10/src/SubmitButton.java:39]: I have all of that lines aligned on my computer. I don't know, why online some lines aren't aligned.

**Review by AndreeaMartinovici — APPROVED** [review]

**Pull request merged**